engine/src/fixedstep/plugin.cpp

```diff
@@ -0,0 +1,24 @@
+#include <cubos/engine/fixedstep/accumulated_time.hpp>
+#include <cubos/engine/fixedstep/plugin.hpp>
+
+using namespace cubos::engine;
+
+const float frametime = 0.01F;
```

Comment:
This should be specified in a resource, so that game devs can configure it or access it if necessary. I would name it `FixedDeltaTime` - it could then be used in fixed-step and physics systems.

That resource would be public, as you did for AccumulatedTime.